packages/blocks/src/api/raw-handling/test/phrasing-content-reducer.js

```diff
@@ -51,6 +51,12 @@ describe( 'phrasingContentReducer', () => {
 		);
 	} );
 
+	it( 'should transform divs to paragraphs', () => {
+		expect(
+			deepFilterHTML( '<div>test</div>', [ phrasingContentReducer ], {} )
```

Comment:
What happens with this testing content?

```
'<div>test<div>this is test</div></div>',
```

Are we going to end up with situations whereby nested divs turn into nested `<p>` tags? I'm not sure nesting is valid.